distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -1791,6 +1786,35 @@ Text.parse_time_of_day self format:Date_Time_Formatter=Date_Time_Formatter.iso_t
 Text.parse_time_zone : Time_Zone ! Time_Error
 Text.parse_time_zone self = Time_Zone.parse self
 
+## ALIAS mid, slice, substring
```

Comment:
Regarding: `slice`

We already _do have_ a `slice` method on `Vector`:
```
    ## GROUP Selections
       Creates a new vector with the skipping elements until `start` and then
       continuing until `end` index.

       Arguments:
       - start: The index of the first element to include.
       - end: The index to stop slicing at.

       > Example
         Remove the first 2 elements then continue until index 5 from the vector.

             [1, 2, 3, 4, 5, 6, 7, 8].slice 2 5 == [3, 4, 5]
    slice : Integer -> Integer -> Vector Any
    slice self start end = @Builtin_Method "Vector.slice"
```

I'm slightly worried that with this alias users can get confused as to what the second argument should be - in `Vector.slice` that is `end` but in `Text.substring` (also aliased by `slice`) it is `length`.

Shouldn't we unify this? I was thinking that `length` argument makes more sense for `substring`, but actually Java's `String::substring` also takes `endIndex` and not `length`.